tests/commands/push.spec.ts

```diff
@@ -323,10 +323,6 @@ describe('balena push: project validation', function() {
 		const expectedErrorLines = [
 			'The --nolive flag is only valid when pushing to a local mode device',
 		];
-		const expectedOutputLines = [
-			'[Warn] "docker-compose.y[a]ml" file found in parent directory: please check',
-			"[Warn] that the correct folder was specified. (Suppress with '--noparent-check'.)",
-		];
```

Comment:
This change and a few others were also made in a PR of mine, #1653, several days older than this PR and still awaiting review... :-)

Reply:
I hadn't realized that addressed type-checking the tests as the description only talks about adding a test case, but I prefer this approach to type-checking as we merge automation/tests type-checking (which should be faster) and inherit from the main tsconfig which means we'll automatically inherit all of the compiler options/etc